packages/expo-image-picker/ios/EXImagePicker/EXImagePicker.m

```diff
@@ -252,20 +252,27 @@ - (void)handleImageWithInfo:(NSDictionary * _Nonnull)info
       [self updateResponse:response withMetadata:metadata];
       completionHandler();
     } else {
-      PHFetchResult<PHAsset *> *assets = [PHAsset fetchAssetsWithALAssetURLs:@[imageURL] options:nil];
-      if (assets.count > 0) {
-        PHAsset *asset = assets.firstObject;
-        PHContentEditingInputRequestOptions *options = [[PHContentEditingInputRequestOptions alloc] init];
-        options.networkAccessAllowed = YES; // Download from iCloud if needed
-        [asset requestContentEditingInputWithOptions:options completionHandler:^(PHContentEditingInput *input, NSDictionary *info) {
-          NSDictionary *metadata = [CIImage imageWithContentsOfURL:input.fullSizeImageURL].properties;
-          [self updateResponse:response withMetadata:metadata];
-          completionHandler();
-        }];
+      PHAsset *asset = nil;
+      if (@available(iOS 11.0, *)) {
+        asset = [info objectForKey:UIImagePickerControllerPHAsset];
       } else {
+        PHFetchResult<PHAsset *> *assets = [PHAsset fetchAssetsWithALAssetURLs:@[imageURL] options:nil];
+        if (assets.count > 1) {
```

Comment:
```suggestion
        if (assets.count > 0) {
```
🤔

Reply:
it doesn't matter, `asset` will be `nil` either way because `assets.firstObject` will be `nil` 😛

Reply:
If I understand the code correctly, we fetch an array of assets and then only assign `asset = assets.first` if `assets.length > 1`. So if `assets.length === 1` (a totally correct scenario) we don't assign to `asset` on iOS < 11

Reply:
@sjchmiela, you're right. It should be `assets.length > 0`.

Reply:
yeah, sorry for my mistake 🤦‍♂